ajax/libs/qartjs/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "qartjs",
+  "version": "1.0.2",
+  "description": "qart.js merge picture and QR code.",
+  "keywords": [
+    "qart",
+    "qr",
+    "qrcode"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kciter/qart.js.git"
+  },
+  "author": "Lee Sun-Hyoup <kciter@naver.com>",
+  "license": "GPL-3.0",
+  "filename": "qart.min.js",
+  "autoupdate": {
```

Comment:
There are 3 releases on NPM and the latest version is newer than GitHub. Let's transfer to use npm auto-update config. Please take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).